app/content-panel/blogs/create/page.tsx

```diff
@@ -1,15 +1,25 @@
+'use client';
+
 /* eslint-disable jsx-a11y/label-has-associated-control */
 
+import React from 'react';
+import { useSession } from 'next-auth/react';
 import styles from './createArticlePage.module.scss';
 import PublishArticleForm from '../../../../components/module-components/blog/PublishArticleForm';
+import { UserRole } from '../../../../Types';
 
 const PostArticle = () => {
+  const { data: session } = useSession();
+
+  // console.log('Session data:', session);
+  const userRole = session?.user.role as UserRole;
+
   return (
     <div className={styles.container}>
       <h2>Објави статија</h2>
 
       <div className={styles.controlsContainer}>
-        <PublishArticleForm />
+        <PublishArticleForm userRole={userRole} />
```

Comment:
The main purpose of the create blog page should be to create/add new blog. On create we should call this endpoint: https://api.learnhub.mk/docs/#content-POSTcontent-blog-posts
We can change the status of already created blogs. That is why you need to send the id of the blog.